projects/chaos/inc/chaos_config.h

```diff
@@ -3,6 +3,8 @@
 #include "gpio.h"
 #include "power_path.h"
 
+#define CHAOS_CONFIG_POWER_PATH_PERIOD_US 250000
```

Comment:
Is there a reason this is in us?

Reply:
That's what the power path currently accepts. In the cleanup PR for that module I'll make it ms.